bin/common.sh

```diff
@@ -121,6 +121,22 @@ JAVA_OPTS+=" ${ZEPPELIN_JAVA_OPTS} -Dfile.encoding=${ZEPPELIN_ENCODING} ${ZEPPEL
 JAVA_OPTS+=" -Dlog4j.configuration=file://${ZEPPELIN_CONF_DIR}/log4j.properties"
 export JAVA_OPTS
 
+if [[ -n "${ZEPPELIN_JMX_ENABLE}" ]]; then
```

Comment:
should it check if `ZEPPELIN_JMX_ENABLE` is set to `false` or something for turning it off?

Reply:
That makes sense. I'll change it

Reply:
Fixed